libs/stocks/feature-shell/src/lib/material/material.module.ts

```diff
@@ -0,0 +1,32 @@
+// material.module.ts
+
+import { NgModule } from '@angular/core';
+import { MatDatepickerModule,
+  MatNativeDateModule,
+  MatFormFieldModule,
+  MatInputModule,
+  MatSelectModule,
+  MatButtonModule
+} from '@angular/material';
+
+@NgModule({
+  imports: [
+    MatDatepickerModule,
+    MatFormFieldModule,
+    MatNativeDateModule,
+    MatInputModule,
+    MatSelectModule,
+    MatButtonModule
+  ],
+  exports: [
+    MatDatepickerModule,
+    MatFormFieldModule,
+    MatNativeDateModule,
+    MatInputModule,
+    MatSelectModule,
+    MatButtonModule
+  ],
+  providers: [ MatDatepickerModule ],
+})
+
+export class MaterialModule {}
```

Comment:
Added new module just for material design stuff. Removed other material designed imports from root module and made this as feature module